easybuild/easyconfigs/o/occt/occt-7.5.0p1-foss-2022a.eb

```diff
@@ -27,10 +27,6 @@ dependencies = [
     ('FreeImage', '3.18.0'),
     ('tbb', '2021.5.0'),
 ]
-
```

Comment:
This would leave no free line between `dependencies` and `configopts`?

Same for other 3 files

Reply:
Not sure how strict we are on this since we do not enforce any linting or pre-commits, besides what the CI static analysis complains about

Reply:
I've restored a blank line in 89eff71.